packages/integrations/markdoc/README.md

```diff
@@ -249,6 +212,83 @@ export default defineMarkdocConfig({
 })
 ```
 
+### Set the root HTML element
+
+Markdoc wraps documents with an `article` tag by default. This can be changed from the `document` Markdoc node. This accepts an HTML element name or `null` if you prefer to remove the wrapper element:
```

Comment:
```suggestion
Markdoc wraps documents with an `<article>` tag by default. This can be changed from the `document` Markdoc node. This accepts an HTML element name or `null` if you prefer to remove the wrapper element:
```
I like tags :sweat_smile: